flutter/lib/src/default_integrations.dart

```diff
@@ -157,6 +157,15 @@ class LoadContextsIntegration extends Integration<SentryFlutterOptions> {
             sdk.addPackage(package['sdk_name']!, package['version']!);
             event = event.copyWith(sdk: sdk);
           }
+
+          // on iOS, captureEnvelope does not call the beforeSend callback,
+          // hence we need to add these tags here.
+          if (event.sdk?.name == 'sentry.dart.flutter') {
```

Comment:
Does this interfer with Androids native integration?

Reply:
no, this integration is iOS only